src/solve.jl

```diff
@@ -127,8 +127,10 @@ function _solve(cp::ControlledPlant,
     for i = 1:NSAMPLES
 
         # simplify the control input for intervals
-        if dim(U₀) == 1
+        if length(U₀) == 1 && dim(U₀) == 1
             U₀ = overapproximate(U₀, Interval)
+        elseif dim(U₀[1]) == 1
+            U₀ = overapproximate.(U₀, Interval)
```

Comment:
I do not understand this part. Above `U₀` is a set and here it is a vector. This will crash in one of those cases.

Reply:
You're right, even if it didn't crash, the code after it would.